operations/pyroscope/helm/pyroscope/templates/ingress.yaml

```diff
@@ -27,6 +27,28 @@ spec:
               number: {{ $.Values.pyroscope.service.port }}
         path: /querier.v1.QuerierService/
         pathType: Prefix
+      - backend:
```

Comment:
An alternate way to this proposal is to enable [Regular Expression Support ](https://kubernetes.github.io/ingress-nginx/user-guide/ingress-path-matching/). With it we could have something like
`path: /(querier.v1.QuerierService|pyroscope/render|pyroscope/render-diff)/`

Reply:
I would like to keep the Path type if possible, as that might be compatible with more than nginx